api/src/Entity/BaseEntity.php

```diff
@@ -46,4 +47,13 @@ abstract class BaseEntity {
     public function getId(): string {
         return $this->id;
     }
+
+    /**
+     * @param BaseEntity $prototype
+     * @param EntityMap  $entityMap
+     */
+    public function copyFromPrototype($prototype, &$entityMap = null) {
```

Comment:
so bekommen User (und künftig Profile) auch diese funktionalität, obwohl sie dort keinen sinn macht.
Kann man aber fürs erste in kauf nehmen

Reply:
habe auf User.php die Methode überschrieben und werfe eine Exception

Reply:
Jetzt musst du das aber auch noch für Profile nachholen, was inzwischen gemerged wurde. Aber wäre es nicht besser, eine Subklasse `CopyableBaseEntity` oder ein Trait dafür zu machen?